e2e/pipelines/test_standard_pipelines.py

```diff
@@ -0,0 +1,220 @@
+    filters = {"source": ["wiki3", "wiki4", "wiki5"]}
+    pipeline = MostSimilarDocumentsPipeline(document_store=document_store)
+    list_of_documents = pipeline.run(document_ids=docs_id, filters=filters)
+
+    assert len(list_of_documents[0]) > 1
+    assert isinstance(list_of_documents, list)
+    assert len(list_of_documents) == len(docs_id)
+
+    for another_list in list_of_documents:
+        assert isinstance(another_list, list)
+        for document in another_list:
+            assert isinstance(document, Document)
+            assert isinstance(document.id, str)
+            assert isinstance(document.content, str)
+            assert document.meta["source"] in ["wiki3", "wiki4", "wiki5"]
+
+
+@pytest.fixture
+def pipeline_yaml_path(tmp_path):
+    path = tmp_path / "pipeline.haystack-pipeline.yml"
+    with open(path, "w") as f:
+        yaml_content = f"""
+            version: ignore
+
+            components:
+            - name: Reader
+                type: FARMReader
+                params:
+                no_ans_boost: -10
+                model_name_or_path: deepset/bert-medium-squad2-distilled
+                num_processes: 0
+            - name: Retriever
+                type: EmbeddingRetriever
+                params:
+                document_store: DocumentStore
+                embedding_model: deepset/sentence_bert
+            - name: DocumentStore
+                type: FAISSDocumentStore
+                params:
+                sql_url: sqlite:///{tmp_path}/faiss_document_store.db
+            - name: PDFConverter
+                type: PDFToTextConverter
+                params:
+                remove_numeric_tables: false
+            - name: TextConverter
+                type: TextConverter
+            - name: Preprocessor
+                type: PreProcessor
+                params:
+                clean_whitespace: true
+            - name: IndexTimeDocumentClassifier
+                type: TransformersDocumentClassifier
+                params:
+                batch_size: 16
+                use_gpu: false
+            - name: QueryTimeDocumentClassifier
+                type: TransformersDocumentClassifier
+                params:
+                use_gpu: false
+
+            pipelines:
+            - name: query_pipeline
+                nodes:
+                - name: Retriever
+                    inputs: [Query]
+                - name: Reader
+                    inputs: [Retriever]
+
+            - name: indexing_pipeline
+                nodes:
+                - name: PDFConverter
+                    inputs: [File]
+                - name: Preprocessor
+                    inputs: [PDFConverter]
+                - name: Retriever
+                    inputs: [Preprocessor]
+                - name: DocumentStore
+                    inputs: [Retriever]
+            """
+        f.write(yaml_content)
+    return path
+
+
+def test_indexing_pipeline_with_classifier():
```

Comment:
`test_indexing_pipeline_with_classifier` and `test_query_pipeline_with_document_classifier` are very similar. They use the same indexing pipeline but two different query pipelines. 
The names of the two tests don't reflect that similarity. Let's change that.